docs/manual_update_control.md

```diff
@@ -57,3 +57,119 @@ To configure a resource to _only_ run when explicitly triggered from the UI, set
 To configure a resource that does _not_ run at start, but still runs whenever a file dependency is changed,
 set `auto_init=False` and `trigger_mode=TRIGGER_MODE_AUTO`. This can be useful for tasks like linting or
 executing tests automatically, for example.
+
+## <span class="pill-tag">Beta</span> Disabling resources through Tilt
+
+Through Tilt’s UI Dashboard, you can enable and disable resources that are available in your Tilt session.
+
+Disabling a resource will stop the running process (if any) and delete any objects owned by that resource. It’s the equivalent of running `tilt down` on a specific resource. Enabling a resource will create, build, deploy, and start any processes for that resource.
+
+> 💡 Disabling and enabling a resource doesn’t take into account any of its resource dependencies. For example, if resource B depends on resource A, disabling resource A will not disable resource B.
+
+> 💡 If you’re using [`tilt args`](tiltfile_config.html) to define a subset of resources Tilt is running, only the resources that are defined through args will show up in the UI Dashboard. In a future release, we’re planning to display all defined resources from the Tiltfile in the UI.
+
+### How to enable the feature
+Disabling resources as a feature is turned off by default. To enable the feature and start using it, you’ll need to be on [Tilt v0.23.5+](https://github.com/tilt-dev/tilt/releases).
+
+Add the following line to your Tiltfile:
```

Comment:
```suggestion
While in beta, the Disable Resources feature requires opting in. First, make sure you're on [Tilt v0.23.5+](https://github.com/tilt-dev/tilt/releases).

Then add the following line to your Tiltfile:
```

1. Make clear the "off by default" behavior is temporary
2. "disabling resources as a feature" is a really awkward noun phrase!
3. add "first," to indicate that there's more coming beyond being on a given tilt version
4. it just seems good to reduce usage of the word "enable" given its multiple meanings in this context

(FWIW, I only felt (1) was really worth opening a comment for, the rest are nits that you can feel free to drop)

Reply:
sounds good! those changes make sense to me.